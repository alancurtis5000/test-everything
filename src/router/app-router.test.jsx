// We're using our own custom render function for redux conneceted compoenentsand not RTL's render
import { render, cleanup } from "../tests/test.utils";
import AppRouter from "./app-router";

describe("AppRouter", () => {
  beforeEach(() => {});

  afterEach(cleanup);

  it("should render", () => {
    const component = render(<AppRouter />);
    expect(component).toBeDefined();
  });

  // it("should be match snapshot", () => {
  //   const component = render(<AppRouter />);
  //   expect(component).toMatchSnapshot();
  // });
});
