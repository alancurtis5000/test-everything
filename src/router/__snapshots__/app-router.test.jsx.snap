// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRouter should be match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="app-content"
      >
        <div>
          Examples Page
          <div
            class="example-counter"
          >
            <h1>
              ExampleCounter
            </h1>
            <div
              class="example-counter-count"
            >
              <h1>
                Count: 
              </h1>
              <h1
                data-testid="count"
              >
                0
              </h1>
            </div>
            <div
              class="example-counter-margin"
            >
              <input
                data-testid="margin"
                id="margin"
                type="number"
                value="1"
              />
            </div>
            <div
              class="button-container"
            >
              <div
                class="example-counter-add"
              >
                <button
                  class="add"
                  data-testid="add"
                >
                  Add
                </button>
              </div>
              <div
                class="example-counter-minus"
              >
                <button
                  class="minus"
                  data-testid="minus"
                >
                  Minus
                </button>
              </div>
              <div
                class="example-counter-reset"
              >
                <button
                  class="reset"
                  data-testid="reset"
                >
                  Reset
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="app-content"
    >
      <div>
        Examples Page
        <div
          class="example-counter"
        >
          <h1>
            ExampleCounter
          </h1>
          <div
            class="example-counter-count"
          >
            <h1>
              Count: 
            </h1>
            <h1
              data-testid="count"
            >
              0
            </h1>
          </div>
          <div
            class="example-counter-margin"
          >
            <input
              data-testid="margin"
              id="margin"
              type="number"
              value="1"
            />
          </div>
          <div
            class="button-container"
          >
            <div
              class="example-counter-add"
            >
              <button
                class="add"
                data-testid="add"
              >
                Add
              </button>
            </div>
            <div
              class="example-counter-minus"
            >
              <button
                class="minus"
                data-testid="minus"
              >
                Minus
              </button>
            </div>
            <div
              class="example-counter-reset"
            >
              <button
                class="reset"
                data-testid="reset"
              >
                Reset
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
